spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py

```diff
@@ -111,6 +113,11 @@ def __init__(
         :param cache_storage_level: (Optional) how to store the cache:
                                     https://sparkbyexamples.com/spark/spark-dataframe-cache-and-persist-explained/.
         :param run_synchronously: (Optional) Run on the Spark master to make debugging easier on dev machines
+        :param sort: (Optional) Whether to sort the df or not. It accepts 2 parameters - {
+                column_for_sorting: "columnName to be used for sorting",
+                drop_column: bool (Whether to drop the column used for sorting)
+                partition_by_column_name: (str) Name of the column that will be used to repartition df
+            }
```

Comment:
partition_by_column_name can an argument on its own.
same for column_for_sorting_within_partition